workflows/qc/markdups-post.wdl

```diff
@@ -18,13 +18,13 @@ workflow markdups_post {
     meta {
         description: "Runs QC analyses which are impacted by duplicate marking"
         outputs: {
-            insert_size_metrics: "`*.txt` output file of `picard collectInsertSizeMetrics`"
-            insert_size_metrics_pdf: "`*.pdf` output file of `picard collectInsertSizeMetrics`"
-            flagstat_report: "`samtools flagstat` report"
-            mosdepth_global_summary: "Summary of whole genome coverage produced by `mosdepth`"
-            mosdepth_global_dist: "Distribution of whole genome coverage produced by `mosdepth`"
-            mosdepth_region_summary: "Summaries of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
-            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
+            insert_size_metrics: "`*.txt` output file of `picard collectInsertSizeMetrics`",
+            insert_size_metrics_pdf: "`*.pdf` output file of `picard collectInsertSizeMetrics`",
+            flagstat_report: "`samtools flagstat` report",
+            mosdepth_global_summary: "Summary of whole genome coverage produced by `mosdepth`",
+            mosdepth_global_dist: "Distribution of whole genome coverage produced by `mosdepth`",
+            mosdepth_region_summary: "Summaries of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`",
+            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`",
```

Comment:
For consistency, we haven't included a trailing comma on the final entry.
```suggestion
            mosdepth_region_dist: "Distributions of coverage corresponding to the regions defined by `coverage_beds` input, produced by `mosdepth`"
```

Reply:
Another good catch.